graph/iterator/resolver_test.go

```diff
@@ -0,0 +1,130 @@
+package iterator_test
+
+import (
+	"testing"
+	"github.com/cayleygraph/cayley"
+	"github.com/cayleygraph/cayley/quad"
+	"context"
+	"github.com/cayleygraph/cayley/graph"
+	"github.com/cayleygraph/cayley/graph/iterator"
+)
+
+func TestResolverIteratorIterate(t *testing.T) {
+	var ctx context.Context
+	qs, err := cayley.NewMemoryGraph()
```

Comment:
Requiring the top-level `cayley` package from low-level `iterators` package is a bit an overkill. Please check other iterator tests, there should be a mock store for such tests.